src/main/scala/org/galaxio/gatling/kafka/actions/KafkaRequestAvro4sActionBuilder.scala

```diff
@@ -4,32 +4,23 @@ import io.gatling.core.action.Action
 import io.gatling.core.action.builder.ActionBuilder
 import io.gatling.core.structure.ScenarioContext
 import io.gatling.core.util.NameGen
-import org.apache.avro.generic.GenericRecord
-import org.apache.kafka.clients.producer.KafkaProducer
 import org.galaxio.gatling.kafka.protocol.KafkaProtocol
 import org.galaxio.gatling.kafka.request.builder.Avro4sAttributes
 
-import scala.jdk.CollectionConverters._
+import scala.reflect.ClassTag
 
-class KafkaRequestAvro4sActionBuilder[K, V](attr: Avro4sAttributes[K, V]) extends ActionBuilder with NameGen {
+class KafkaRequestAvro4sActionBuilder[K: ClassTag, V: ClassTag](attr: Avro4sAttributes[K, V])
```

Comment:
As far as I remember, ClassTags are not needed because avro4s will be able to derive Serdes without it. And there was an idea to unify all these things with byte arrays. We need to pass already serialized values ​​to builders and work with them the same way as with others